libs/books/feature/src/lib/book-search/book-search.component.spec.ts

```diff
@@ -18,6 +28,14 @@ describe('BookSearchComponent', () => {
         NoopAnimationsModule,
         MatProgressSpinnerModule,
         SharedTestingModule
+      ],providers: [
+        provideMockStore({
+          initialState: {
```

Comment:
Alternatively, you can pass `selectors` to override individual selectors used by the component.